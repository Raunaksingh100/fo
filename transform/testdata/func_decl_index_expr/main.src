package main

func A[T] (x []T) {
	print(x[0])
}

func main() {}
