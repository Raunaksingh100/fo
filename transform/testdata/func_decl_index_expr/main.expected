package main

import "reflect"

func A(T interface{}, x interface{}) {
	print(reflect.ValueOf(x).Index(0).Interface())
}

func main() {}
